### ISSUE TITLE
Storefront and Google Analytics by Yoast in multisite

### ISSUE THREAD

**Issue by steve-chater** [Yoast/google-analytics-for-wordpress#278]
Hi.  Thought I'd let you know of an issue I came across when trialling the WooTheme theme Storefront (I already had Travelify and Mystile installed and the problem didn't occur until I installed Storefront). 

I'm trying Storefront in a multisite test environment. When customising the theme the menu displays on the left but the main screen repeatedly flashes the WP logo and says "You have logged in successfully". If I make a change via the menu (eg Site Title) and click "Save & Publish" the message "Cheating, are we?" is displayed and the changes are not saved.

I turned off all plugins and re-introduced until striking the problem when Google Analytics by Yoast (Version 5.2.8) was Network Activated. Note that the problem was not encountered if the plugin is activated on each individual site.

**Comment by CarolineGeven** [comment]
Thanks for reporting this issue. 
Unfortunately we do not have enough information to reproduce this problem. Please answer the following questions so we can try to figure out what is going on and help us reproduce it.

* What version of WordPress are you running?
* What version of PHP are you running?
* What kind of webserver are you running (Apache, Nginx,IIS)?
* Are you running the latest version of the plugin?
* Do you still have this issue if you switch to the default WordPress theme (*e.g. Twenty Fourteen*) and deactivate all other plugins?

Also, it will help us a lot if you can provide a backtrace of the error encountered. [This gist](https://gist.github.com/jrfnl/5925642) will explain how to enable the backtrace in your website: https://gist.github.com/jrfnl/5925642

Thank you.

**Comment by steve-chater** [comment]
Hi Caroline

I'm sorry to say that I've moved on to use another theme (rather than
Storefront) and am not in a position to recreate the problem again.

What I can tell you is:
- I am using Wordpress 4.1
- PHP Version 5.3.6
- running MAMP (ie Apache on mac)
- running Version 5.2.8 of Google Analytics by Yoast (the latest version I
believe)
- when I first had the problem I pulled from backup and found that I did
NOT have the problem with any of my other themes (I checked with Twenty
Twelve, Travelify and Mystile).  It was only after again installing
Storefront that the problem reappeared, and then it occurred no matter
which theme I activated.

Sorry I can't give you the backtrace, but I hope the above helps.

Steve Chater
